src/WasmSdk/Sdk/Sdk.props

```diff
@@ -16,14 +16,16 @@ Copyright (c) .NET Foundation. All rights reserved.
     <!-- WASM projects defaults to browser-wasm -->
     <RuntimeIdentifier Condition="'$(RuntimeIdentifier)' == ''">browser-wasm</RuntimeIdentifier>
     <UseMonoRuntime>true</UseMonoRuntime>
-    
     <SelfContained>true</SelfContained>
     <PublishTrimmed Condition="'$(PublishTrimmed)' == ''">true</PublishTrimmed>
 
+    <!-- Disable accelerated builds in Visual Studio -->
+    <AccelerateBuildsInVisualStudio>false</AccelerateBuildsInVisualStudio>
```

Comment:
I'm curious if we can set the property in the SDK (nuget) pack. We are trying to keep most of the settings there

Reply:
If you want to push this down to the package, I'm fine if we do that. Do you want me to close this PR so you can open one in the runtime repo (I imagine this is where the package lives)